### DISCUSSION TITLE
Zoom for camera

### DISCUSSION THREAD

**Discussion by johny77g** [mebjas/html5-qrcode#305] [Ideas]
Hi, is it possible to add a camera zoom option ? Something like:

if ('zoom' in capabilities) {
......
slider_zoom = .........
track.applyConstraints({advanced: [ {zoom: slider_zoom } ]});
}

**Comment by AntonDr** [comment]
Any updates?

**Reply by mebjas** [reply]
Feature request for zoom feature: https://github.com/mebjas/html5-qrcode/issues/330
